README.md

```diff
@@ -10,14 +10,21 @@ FairScale is a PyTorch extension library for high performance and large scale tr
 
 FairScale supports:
 * Parallelism:
-   * pipeline parallelism (fairscale.nn.Pipe)
+   * Pipeline parallelism (fairscale.nn.pipe)
+   * Asynchronous Pipeline parallelism (fairscale.nn.async_pipe)
+   * Mixture of experts (fairscale.nn.moe.moe_layer)
+   * Model Parallelism (fairscale.nn.model_parallel.layers)
+   * _experimental_ AmpNet (fairscale.experimental.nn.ampnet_pipe)
 * Sharded training:
-   * Optimizer state sharding (fairscale.optim.oss)
-   * Sharded grad scaler - automatic mixed precision
-   * Sharded distributed data parallel
+   * Optimizer state sharding (fairscale.optim.OSS)
+   * Sharded grad scaler - automatic mixed precision (fairscale.optim.grad_scaler)
+   * Sharded distributed data parallel (fairscale.nn.ShardedDataParallel)
+   * Fully Sharded Data Parallel (FSDP) (fairscale.nn.FullyShardedDataParallel)
+   * _experimental_ Offload (fairscale.experimental.nn.offload)
```

Comment:
One thing to note is that this is not part of the pip package yet. Should we hold off mentioning this until the next release? I was going to add release notes and an usage example.

Reply:
yes... let's add it with next release